tutorials/inference.md

```diff
@@ -123,3 +131,16 @@ litgpt generate_tp meta-llama/Llama-2-70b-chat-hf \
 | 8       | 6.15 GB     | 6.20      |
 
 Smaller devices can also be used to run inference with this technique.
+
+&nbsp;
+## Use FlashAttention 3
+
+[FlashAttention](https://github.com/Dao-AILab/flash-attention) is a highly optimized technique for computing the attention mechanism, which is designed to be fast and memory-efficient while maintaining exact computational precision. It specifically addresses the I/O bandwidth and memory access patterns to minimize latency and maximize throughput on modern GPU architectures. LitGPT uses FlashAttention v2 automatically for modern GPUs. 
+
+To use the latest FlashAttention v3 implementation, which is optimized for Hopper GPUs, such as H100 and H800, install the `flash-attention` package via
+
+```bash
+pip install git+https://github.com/Dao-AILab/flash-attention.git
+```
```

Comment:
It seems from the fa repo that in oder to install fa3 one need to first cd into the `hopper` [folder](https://github.com/Dao-AILab/flash-attention/tree/main/hopper) and then install.  Did you had to do it as well ? Or did the install here worked ?

(asking bc I am trying to add fa3 to my litgpt based repo)